src/model.ts

```diff
@@ -24,10 +24,10 @@ export class Model {
         ? animal.abundance[this.environment].notSunny
         : animal.abundance[this.environment].sunny;
       const animalAbundance = getRandomInteger(abundanceRange.min, abundanceRange.max);
-      // TODO: for now adjust each abundance number by 30% if it is less sunny, ultimately this should be in abundanceRange
-      const animalAbundanceAdjusted = Math.floor(this.sunnyDayFequency === 0 ? animalAbundance : animalAbundance * .7);
-      for (let x = 0; x < animalAbundance; x++) {
-        // TODO: for now spread spawn over max time, but this should be more realistic
+      // increase each abundance number by 20% if it is less sunny
+      // ideally this should be in abundanceRange, but we never received taxa specific details
+      const animalAbundanceAdjusted = Math.floor(this.sunnyDayFequency === 0 ? animalAbundance * 1.2 : animalAbundance);
```

Comment:
```suggestion
      const animalAbundanceAdjusted = Math.floor(this.sunnyDayFequency === 0 ? animalAbundance * kLowSunAbundanceFactor : animalAbundance);
```
or whatever the constant is called.

Reply:
Ah, I see.  I'll add one more small PR and tweak this.